webtau-http/src/main/java/org/testingisdocumenting/webtau/http/validation/HttpValidationResult.java

```diff
@@ -197,6 +202,11 @@ public long getElapsedTime() {
         Map<String, Object> result = new LinkedHashMap<>();
 
         result.put("id", id);
+
+        if (!personaId.isEmpty()) {
```

Comment:
Should we pass the whole `Persona` to `HttpValidationResult` so that this check here can be `!persona.isDefault()` so we don't propagate knowledge of what ID constitutes the default persona?  At a minimum I think this should compare to the `DEFAULT_PERSONA_ID` constant rather than checking it's empty.

Reply:
You are right. On report side I still rely on it to be empty though. Adding a change. 
I read it rather as "persona is not set" vs "persona is default". Well I read it both ways depending on the context 🤷‍♂️